packages/app/src/app/pages/SignIn/Onboarding.tsx

```diff
@@ -8,11 +8,13 @@ import { StyledButton } from 'app/components/dashboard/Button';
 import track from '@codesandbox/common/lib/utils/analytics';
 
 const ROLE_OPTIONS = [
-  { value: 'frontend', label: 'Frontend developer' },
-  { value: 'backend', label: 'Backend developer' },
-  { value: 'fullstack', label: 'Fullstack developer' },
+  { value: 'frontend', label: 'Front-End Developer' },
+  { value: 'backend', label: 'Back-End Developer' },
+  { value: 'fullstack', label: 'Full-Stack Developer' },
+  { value: 'designer', label: 'Lead Developer' },
+  { value: 'designer', label: 'Engineering Manager' },
```

Comment:
I think there's a wrong value here for Lead Developer and Engineering Manager, they shouldn't have value `designer` right?

Reply:
True. Not sure how I missed that. Just committed an update. Can you have a look @alexnm ?